### DISCUSSION TITLE
How to make hybrid assembly of a viral genome

### DISCUSSION THREAD

**Discussion by sekhwal** [ablab/spades#882] [Q&A]
Hi, I am looking to make Hybrid assembly of a viral genome. I have got paired-end reads from Illumina and long reads from MinION from a viral sample. Please suggest if I can use SPAdes for viral hybrid genome assembly. Also, please let me know if I need to trim the data before making assembly.

Thank you!

**Comment by asl** [comment]
What kind of the data you're having? DNA virus? RNA virus? metavirome? transcriptome?

**Reply by sekhwal** [reply]
I ran SPAdes with the following command. However, in the output file (scaffold.fasta and contigs.fasta), it is showing several scaffolds and contigs. However, the purpose of making assembly is that I am looking to make an assembly of the whole viral genome, See (NC_001802.1) for example.

command: spades.py -k 21,33,55,77 --careful -1 file_R1_001.fastq.gz -2 file_R2_001.fastq.gz --nanopore merge.fastq.gz -o out_spades
